File: src/bots/redirect_help.php
<?php

namespace WpRefs\Bots\Redirect;
/*
usage:
use function WpRefs\Bots\Redirect\page_is_redirect;
*/

function page_is_redirect($title, $text)
{
    // #пренасочване
    // ---
    if (preg_match('/^#(пренасочване|redirect)/', $text)) {
        return true;
    }
    // ---
    return false;
}

File: python/src/bots/redirect_help.py
"""
Redirect helper module

Usage:
    from src.bots.redirect_help import page_is_redirect
"""

import re


def page_is_redirect(title, text):
    """
    Check if page is a redirect
    
    Args:
        title: Page title
        text: Page text
        
    Returns:
        True if page is redirect
    """
    # Check for redirect patterns
    if re.match(r'^#(пренасочване|redirect)', text, re.IGNORECASE):
        return True
    
    return False
